src/main/java/com/project/poorlex/exception/battleuesr/BattleUserCustomException.java

```diff
@@ -0,0 +1,25 @@
+package com.project.poorlex.exception.battleuesr;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public class BattleUserCustomException extends RuntimeException{
```

Comment:
`RuntimeException 뒤에 띄어쓰기 빠져보입니다!
```suggestion
public class BattleUserCustomException extends RuntimeException {
```